packages/msw/mixin.browser.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-env browser */
+
+import { Mixin } from 'hops-mixin';
+
+const registerBrowserMock = (
+  { graphql, rest },
+  { type, method, identifier, data }
+) => {
+  if (type === 'graphql') {
+    return graphql[method](identifier, (_, res, ctx) => res(ctx.data(data)));
+  }
+
+  if (type === 'rest') {
+    return rest[method](identifier, (_, res, ctx) =>
+      res(
+        ctx.status(200),
+        typeof data === 'undefined' ? undefined : ctx.json(data)
+      )
+    );
+  }
+};
+
+class MswMixin extends Mixin {
+  async bootstrap() {
+    if (process.env.ENABLE_MOCKS === 'true') {
```

Comment:
Here we should probably use `this.config` instead and map the env var `ENABLE_MOCKS` to a config value, so that we can test any bundle.